src/libraries/System.Text.Json/System.Text.Json.SourceGeneration.Tests/JsonSourceGeneratorTests.cs

```diff
@@ -2,16 +2,32 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
 using Xunit;
 
 namespace System.Text.Json.SourceGeneration.Tests
 {
-    public class JsonSerializerSouceGeneratorTests
+    public class JsonSerializerSourceGeneratorTests
```

Comment:
I'd expect a more robust test here in terms of having members on the test classes, similar to what you have in the unit test project. Specifically we should have public and non-public properties (with (non-)public getters/setters), fields, and constructors. Also we should have usages for all the attributes that System.Text.Json ships.

Reply:
Will create current json serializer attributes in next pr.